.github/workflows/check_doxygen_awesome_version.yml

```diff
@@ -2,6 +2,9 @@ name: Auto-update doxygen-awesome
 
 on:
   workflow_dispatch:
+  push:
+    branches:
+    - master
```

Comment:
I'd like to suggest triggering the workflow nightly, e.g. like this:

    schedule: 
      - cron: 0 2 * * *

Advantages: 
  - fires at a constant rate, independent of how active development is at that point.
  - only triggers once a day. If the the CI breaks, we have time to react.
  - wont be a condition for other PRs to succeed.

Reply:
Good idea. But I think that checking one a week it sufficient.